src/view/wallet/Receive.svelte

```diff
@@ -11,6 +11,12 @@
     })
 </script>
 
+<style>
+    img {
+        width: 100%;
```

Comment:
Maybe we have to change this when we get the final designs. Makes sense to have the address in the bottom or maybe on the second page, in order to check, if it is the right one. But certainly better for scanning.